### ISSUE TITLE
Non-standard WP_Thumb::error() missing in official release

### ISSUE THREAD

**Issue by entr** [fooplugins/foogallery#93]
I'm using an external provider for wp-thumb which always worked fine since FooGallery has the proper `class_exists()` checks but now when I updated to 1.2.15 I am getting this fatal

```
PHP Fatal error:  Uncaught Error: Call to undefined method WP_Thumb::error() in /var/www/web/app/plugins/foogallery/includes/class-thumbnails.php:77
Stack trace:
#0 /var/www/web/app/plugins/foogallery/includes/admin/class-admin-notices.php(64): FooGallery_Thumbnails->run_thumbnail_generation_tests()
#1 /var/www/web/wp/wp-includes/plugin.php(524): FooGallery_Admin_Notices->display_thumb_test_notice('')
#2 /var/www/web/wp/wp-admin/admin-header.php(250): do_action('admin_notices')
#3 /var/www/web/wp/wp-admin/edit.php(299): require_once('/var/www/web/wp...')
#4 {main}
  thrown in /var/www/web/app/plugins/foogallery/includes/class-thumbnails.php on line 77
```

I don't know what wp-thumb fork is shipped with FooGallery but can you make it so it doesn't break if stable(official) wp-thumb from plugins.wordpress.org is used instead? In this specific situation the `errored()` method could be used followed by `$this->error` for the actual error.

**Comment by bradvin** [comment]
This should be resolved in develop branch with 338eccdfb8a42e62e3332fd71afe09f3c8989e8f

**Comment by entr** [comment]
Thank you!
